Screens/Settings.tsx

```diff
@@ -0,0 +1,29 @@
+import { View, Text, TouchableOpacity } from "react-native";
+import { useContext } from "react";
+import { useNavigation } from "@react-navigation/native";
+import MadeButton from "../components/button";
+import Header from "../components/header";
+import ThemeContext from "../theme/ThemeContext";
+import { LightStyles, DarkStyles } from "../Styles";
+
+
+//needs style on #18
```

Comment:
?? What does this mean? Is this comment even needed?

Reply:
this is from the previous pr